content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -32,19 +32,24 @@ mitigate some future vulnerabilities too.
 {{% thirdparty-content single="true" %}}
 <!-- if adding another runtime in the future, omit the single setting -->
 
-This is a Linux only feature. In addition, support is needed in the 
+This is a Linux only feature. In addition, support is **needed** in the 
 {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
 to use this feature with Kubernetes stateless pods:
 
-* CRI-O: v1.25 has support for user namespaces.
+* CRI-O: supports user namespaces with all Kubernetes versions >= v1.25
```

Comment:
```suggestion
* CRI-O: version 1.25 (and later) of CRI-O supports user namespaces for containers.
```

Using CRI-O v1.24 and Kubernetes v1.26? It won't work (AIUI). It's the CRI-O version that matters here.

Reply:
I can change it to that, but CRIO is released in lockstep with k8s. You can't use CRIO 1.24 with k8s 1.26 anyways, AFAIK.